flycheck.el

```diff
@@ -6556,6 +6556,7 @@ See URL `http://cppcheck.sourceforge.net/'."
             (option-list "-I" flycheck-cppcheck-include-path)
             (option-list "--std=" flycheck-cppcheck-standards concat)
             (option-list "--suppress=" flycheck-cppcheck-suppressions concat)
+            (option "--suppressions-list=" flycheck-cppcheck-suppressions-file concat)
```

Comment:
Do we have a `flycheck-cppcheck-suppressions-file` variable?  I'm not seeing any.  Am I missing something, or do you just `setq` the variable in your config?

Reply:
Oh, I had it in dir-locals.el so I didn't notice it missing; added in 5ce24713c8f00b42fa3286c10b5951fe4f114a39